charts/telemetry-api/values-prod.yaml

```diff
@@ -12,6 +12,7 @@ env:
   TOKEN_EXCHANGE_JWK_KEY_SET_URL: http://dex-roles-rights-prod.prod.svc.cluster.local:5556/keys
   TOKEN_EXCHANGE_ISSUER_URL: https://auth-roles-rights.dimo.zone
   VEHICLE_NFT_ADDRESS: '0xbA5738a18d83D41847dfFbDC6101d37C69c9B0cF'
+  MANUFACTURER_NFT_ADDRESS: '0x3b07e2A2ABdd0A9B8F7878bdE6487c502164B9dd'
```

Comment:
Double-checked with the [deploy list](https://github.com/DIMO-Network/web3-deploys?tab=readme-ov-file), these look right. 👍